.travis.yml

```diff
@@ -139,8 +140,10 @@ script:
         /tmp/julia/bin/julia-debug --sysimage-native-code=no -e 'true'
     - /tmp/julia/bin/julia -e 'versioninfo()'
     - pushd /tmp/julia/share/julia/test
-    - /tmp/julia/bin/julia --check-bounds=yes runtests.jl $TESTSTORUN &&
-      /tmp/julia/bin/julia --check-bounds=yes runtests.jl LibGit2/online OldPkg/pkg Pkg/pkg download
+    # skip tests if only files within the "doc" dir have changed
+    - if [ ! -z $(git diff --name-only $TRAVIS_COMMIT_RANGE | grep -v '^doc/') ]; then
```

Comment:
Note: `git diff --name-only` always shows the file path relative to the root of the directory even when you are within a subdirectory.